packages/application-shell/src/components/application-shell/application-shell.js

```diff
@@ -285,13 +285,13 @@ export const RestrictedApplication = props => (
                                 render={props.render}
                               />
                               {/* Project routes */}
-                              {/* Redirect from base project route to dashboard */}
+                              {/* Redirect from base project route to welcome */}
                               <Route
```

Comment:
If we do that, I would simply remove this route. What will happen then is that the fallback app (which most likely is the application served when requests are e.g. `/` or `/:projectKey`) will handle the redirect `/:projectKey` -> `/:projectKey/welcome`.

I would avoid hardcoding any more routes here, who know what will come in the future.

Reply:
Makes sense. I am confused by apparently two opinions towards this:

1. Keep it backwards compatible
2. Value in removing the route here entirely

I don't mind and am really open to what the best solution is. What would you prefer?

Reply:
Well, eventually we should remove it. It's just the question if we want to do it now or maybe wait for the "next major" release?

Reply:
When would be the next major release. Did we plan one? I personally don't mind major releases now as custom apps are in beta. Just my take though.

Reply:
Probably #293 but I haven't planned yet when to work on that. So, yeah I'm fine with the breaking change now.